src/views/node/HistoPheno.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <!-- status -->
+  <AppStatus v-if="isLoading" code="loading"
+    >Loading histo-pheno data</AppStatus
+  >
+  <AppStatus v-else-if="isError" code="error"
+    >Error loading histo-pheno data</AppStatus
+  >
+
+  <!-- results -->
+  <Apex v-if="series.length" type="bar" :options="options" :series="series" />
+</template>
+
+<script setup lang="ts">
+import { useRoute } from "vue-router";
+import Apex from "vue3-apexcharts";
+import { getHistoPheno } from "@/api/histo-pheno";
+import { Node } from "@/api/node-lookup";
+import { useQuery } from "@/util/composables";
+import { computed, watch } from "vue";
+import theme from "@/global/variables.module.scss";
+
+/** route info */
+const route = useRoute();
+
+interface Props {
+  /** current node */
+  node: Node;
+}
+
+const props = defineProps<Props>();
+
+/** chart options */
+const options = computed(() => ({
+  colors: [theme.theme],
+  chart: {
+    type: "bar",
+    redrawOnParentResize: true,
+    width: "100%",
+    height: "100%",
```

Comment:
When you download the chart as an SVG, PNG, or the backing CSV, you currently get a random filename. It'd be nice to override the filename to include the phenotype at least for the CSV, as it's not present in the file itself. You can add the `toolbar.export.(svg/png/csv).filename` path to this `chart` dictionary to accomplish that, as documented here under the "export" section: https://apexcharts.com/docs/options/chart/toolbar/.

Reply:
Well, now they're all named "histo-pheno", which is an improvement, but it'd be better if it incorporated the node name in the filename somehow IMHO (at least for the CSV, where there's no indication of the node name in the file's contents itself). Perhaps you could add `${props.node.name}` as a suffix to the ID?

Reply:
I tried that, but apex messed with the filename for some reason. It added X's in place of spaces. It might be trying to make sure there's no whitespace in the filename (although it still left some).

I didn't want to spend too much time on this because it's not in 2.0 and I'm not sure if anyone will even want the feature. I was considering just removing the download bar all together.

Reply:
Ah, alright, sounds good.